packages/data-transport-layer/src/utils/eth-tx.ts

```diff
@@ -1,9 +1,18 @@
 /* Imports: External */
 import { ethers } from 'ethers'
 
+// This is expected to return 0 or 1
 export const parseSignatureVParam = (
   v: number | ethers.BigNumber,
   chainId: number
 ): number => {
-  return ethers.BigNumber.from(v).toNumber() - 2 * chainId - 35
+  if (typeof v === 'object') {
+    v = ethers.BigNumber.from(v).toNumber()
+  }
+  // Handle unprotected transactions
+  if (v === 27 || v === 28) {
+    return v - 27
```

Comment:
Just need to be confident that this doesn't allow someone to play ETH transactions on L2

Reply:
By not having a chainid, the transaction can be replayed. So any transactions sent to L1 without a chainid are subject to being replayed on L2, assuming the account on L2 has the correct prestate (nonce/value to cover gas)